src/adv.py

```diff
@@ -49,3 +54,45 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+while game_state is 1:
+    print(player.room)
+    x = input("Which direction would you like to go? ")
+    if x == 'n':
+        if hasattr(player.room, 'n_to'):
+            cls()
+            player.room = player.room.n_to
+        else:
+            cls()
+            print("You cannot go that way.")
+
+    elif x == 'e':
+        if hasattr(player.room, 'e_to'):
+            cls()
+            player.room = player.room.e_to
+        else:
+            cls()
+            print("You cannot go that way.")
+
+    elif x == 's':
+        if hasattr(player.room, 's_to'):
+            cls()
+            player.room = player.room.s_to
+        else:
+            cls()
+            print("You cannot go that way.")
```

Comment:
I was thinking there was a way you could make this DRYer (Do not Repeat Yourself), but now I'm not so sure because my code does something similar.